src/test/java/seedu/address/logic/commands/SelectCommandTest.java

```diff
@@ -0,0 +1,78 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.ModelManager;
+import seedu.address.testutil.TypicalEvents;
+
+
+
+public class SelectCommandTest {
+
+    @Test
+    public void constructor_nullIndex_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new SelectCommand(null));
+    }
+
+    @Test
+    public void execute_nullModel_throwsNullPointerException() {
+        ModelManager model = new ModelManager();
+        model.setEventBook(TypicalEvents.getTypicalEventBook());
+        assertThrows(NullPointerException.class, () -> new SelectCommand(Index.fromZeroBased(0)).execute(null));
+    }
+    @Test
+    public void event_index_outOfRange() {
+        ModelManager model = new ModelManager();
+        model.setEventBook(TypicalEvents.getTypicalEventBook());
+        assertThrows(CommandException.class, () -> new SelectCommand(Index.fromZeroBased(100)).execute(model));
+    }
+
+    @Test
+    public void equals_same_index() {
```

Comment:
better if this could folow the naming conventions 
Underscores may be used in test method names using the following three part format `featureUnderTest_testScenario_expectedBehavior()`
but i am not sure if they are this strict in grading